package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ssb-keys-mnemonic",
   "description": "Module that converts from/to SSB keys and BIP39 mnemonic codes",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I usually do a separate commit for the version. Could you revert this one?

Reply:
Done.